### DISCUSSION TITLE
Adding `Equatable` for direct comparison of `TinyColor`s

### DISCUSSION THREAD

**Discussion by calvintam236** [TinyCommunity/tinycolor2#11] [Ideas]
Do you think we should extends `TinyColor` from `Equatable`? Looks like a good way to compare between `TinyColor`s.

Ref: https://pub.dev/packages/equatable

**Comment by dsyrstad** [comment]
I'd rather not bring in other dependencies.

**Reply by calvintam236** [reply]
Alternatively, we can do the hard code way (operator `==`) for comparison. This way does not require any dependencies, just longer and uglier.

**Reply by dsyrstad** [reply]
I use TinyColor for color manipulation, after that it goes back into a `Color` so I never need to check if they're equal. I think we should keep TinyColor "tiny" and focused. Let's fix the bugs, get it to null-safety, and publish it to pub.dev.

**Reply by calvintam236** [reply]
https://github.com/bgrins/TinyColor#color-utilities

This functionality is part of the original `tinycolor2` and was not ported over. This addition would be easy and simple.

**Reply by calvintam236** [reply]
Update: PR #16 implements this feature without any new dependency.
